target/linux/brcm63xx/image/bcm63xx.mk

```diff
@@ -207,6 +207,19 @@ endef
 TARGET_DEVICES += R1000H
 
 ### ADB ###
+define Device/AGPWI
+  $(Device/bcm63xx)
+  DEVICE_VENDOR := ADB
+  DEVICE_MODEL := P.DG A4001N A-000-1A1-AX (AGPWI)
```

Comment:
Please use the DEVICE_ALT0_VENDOR/DEVICE_ALT0_MODEL syntax for the name in brackets.

Reply:
I corrected it. Is it ok now?